os/drivers/audio/alc5658.c

```diff
@@ -832,7 +814,7 @@ static int alc5658_pause(FAR struct audio_lowerhalf_s *dev)
 		/* Disable interrupts to prevent us from suppling any more data */
 
 		priv->paused = true;
-		alc5658_setvolume(priv, priv->volume, true);
+		alc5658_setvolume(priv);
```

Comment:
Mute is needed. for below case.
Pre-status : volume level is 20 and there are 2 songs in track
a.mp3 finished -> set mute to reduce noise -> unmute and play b.mp3

Reply:
Yes, Mute is definitely needed. But, mutes changes in many other places are also missing. Will add & fix mute in another commit.